source/includes/_intro.md

```diff
@@ -394,25 +394,267 @@ The first step in most situations will be to instantiate a client. (TODO these c
+A customer may change their mind about the number or type of seats to be
+reserved for a performance. If your customer selects and reserves seats, then
+goes back to select and reserve additional seats or removes the reserved seats
+from their basket, the initial seats should be [released](#release).
+
+Failure to release the initial reservation will mean the customer is unable to
+reserve seats (even though they are actually available). Ingresso will
+automatically release any reserve after 15 minutes of inactivity, but you must
+release seats as soon as possible. Ingresso reserves the right to disable the
+API account of any partner that does not release tickets when they should be.
+
+**Representative Test:** User makes a reservation for any ticket then clicks
+"back" or otherwise moves away from the reservation confirmation screen<br/>
+When the customer moves away from the reservation screen<br/>
+Then your application should call the `release` resource of the
+Ingresso API. This must be done before any further call to `reserve`.<br/>
+Optionally it should also display a message to the customer indicating the
+seat reservation has been released.
+
+### Display friendly error messages during purchase
+When purchasing on the Internet, it can be very disconcerting for customers
+when errors occur. Make sure any errors are shown to the customer, with a
+message that their card has not been charged.
+
+**Representative Test:** User makes a reservation for any ticket type on event 7AB, and
+at purchase enters a postcode of `W5 4JJ`.<br/>
+When the customer enters a postcode of `W5 4JJ` in the address details of the
+purchase confirmation,<br/>
+Then the Ingresso API will generate a booking failure and return the
+result. The application should show the customer that an error occurred, the
+reservation was not successful and that their card has not been charged.
+
+### Display different send methods including eTickets
+Various venues support different methods of sending tickets (including post,
+pickup from box office, or printable eTickets). Some of these may carry an
+additional fee. Where available, different send methods should be offered to
+the customer, and your application should support eTickets (either provided by
+Ingresso or your own format with a barcode retrieved from the Ingresso API).
+
+**Representative Test:** User confirms reservation of tickets for Toy Story -- The Opera
+(7AA)<br/>
+When the customer selects the Toy Story(7AA) event<br/>
+Then the available send methods of *Printable eTicket* and *Post worldwide* are
+shown.
+
+**Representative Test:** User selects user printable tickets and checks out<br/>
+When the customer has chosen the printable ticket (eTicket) option<br/>
+Then the ticket with the appropriate barcode will be sent to their email or
+otherwise permanently saved within your application after purchase has been
+confirmed
+
+### Gracefully handle reservation failures and seat changes
```

Comment:
Maybe the example from the "Display friendly error messages during purchase" section should be added in to here instead?